src/components/HathorSelect.js

```diff
@@ -27,7 +27,7 @@ import { ReactComponent as RowDown } from '../assets/images/arrow-down-nav-dropd
  *   name: <string>          // Display name for the current selected option.
  * }
  */
-const HathorSelect = ({ options, onSelect, value }) => {
+const HathorSelect = ({ options, onSelect, value, background }) => {
```

Comment:
issue: The way this is implemented, this parameter is mandatory. We should also adjust every other place where this component is called, or make this parameter optional.

Also, we need to add it to the docstrings.

Reply:
Property usage fixed. Waiting review. 🙂